src/main/java/org/jabref/gui/fieldeditors/contextmenu/EditorMenus.java

```diff
@@ -1,30 +1,23 @@
 package org.jabref.gui.fieldeditors.contextmenu;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.function.Supplier;
-
-import javax.swing.AbstractAction;
-import javax.swing.Action;
-
-import javafx.scene.control.CustomMenuItem;
-import javafx.scene.control.Label;
-import javafx.scene.control.MenuItem;
-import javafx.scene.control.SeparatorMenuItem;
-import javafx.scene.control.TextArea;
-import javafx.scene.control.Tooltip;
-
+import javafx.scene.control.*;
 import org.jabref.gui.actions.CopyDoiUrlAction;
-import org.jabref.gui.fieldeditors.EditorTextArea;
+import org.jabref.logic.formatter.bibtexfields.CleanupURLFormatter;
 import org.jabref.logic.formatter.bibtexfields.NormalizeNamesFormatter;
 import org.jabref.logic.l10n.Localization;
 
+import javax.swing.*;
```

Comment:
Please no general imports with a star, because that loads all classes in the package. 
Our code formatter for eclipse and intellij should have the correct orders